_posts/api-fortress-integration

```diff
@@ -0,0 +1,26 @@
+---
+title: Apiary and API Fortress Announce Partnership
+excerpt: Apiary is embracing the CommonMark Markdown syntax for API description.
+layout: post
+date: 2016-11-08 10:00:00 +0200
+author: natalie
+published: true
+---
+
+## API Fortress announces successful integration with Apiary
+
+Partnership Integrates API Fortress Monitoring and Testing
+with Apiary’s API-First Design Platform
+
+New York, NY — November 7, 2016 — API Fortress, a leading provider of API monitoring and test solutions, and Apiary, the leader in API-First design platforms, today announced new integration capabilities. The joint functionality provides mutual customers the ability to link the start and end of the API development lifecycle. Use API design contracts to generate tests for test-driven development and continuous testing in production.
+Test-driven development for APIs has traditionally been hard to achieve. Now, you can login to your Apiary account from within API Fortress and automatically generate tests from your Apiary Blueprints. The tests generated can be used against both staging and production environment and connects the API-First development power of Apiary with API Fortress’s robust monitoring and testing.
+The joint functionality will be available on all API Fortress accounts, and the Apiary API Flow Pro plan.
+
+Free trials are available with full functionality are available on both [http://www.apifortress.com] and [http://www.apiary.io].
```

Comment:
Link syntax should be different https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

```diff
- Free trials are available with full functionality are available on both [http://www.apifortress.com] and [http://www.apiary.io].
+ Free trials are available with full functionality are available on both [apifortress.com](http://apifortress.com) and [apiary.io](https://apiary.io).
```